lab_1_classify_by_unigrams/main.py

```diff
@@ -84,12 +117,33 @@ def compare_profiles(
     In case of corrupt input arguments or lack of keys 'name' and
     'freq' in arguments, None is returned
     """
+    if "name" in unknown_profile and "freq" in unknown_profile and "name" in profile_to_compare and "freq" in profile_to_compare:
```

Comment:
if not - `return None` - try early return strategy

Reply:
done